timApp/static/scripts/tim/answer/answerbrowser3.ts

```diff
@@ -1735,6 +1731,9 @@ export class AnswerBrowserController
             return;
         }
         this.taskInfo = r.result.data;
+        if (r.result.data.modelAnswer) {
+            this.modelAnswer = r.result.data.modelAnswer;
+        }
```

Comment:
Hmm, onko tässä `this.onlyValid = false;`:n poisjättäminen ihan tahallista? Eli tämän PR:n myötä invalidit vastaukset ei näytetä aina oletuksella `modelAnswer`-tehtävissä?